include/rocksdb/utilities/transaction.h

```diff
@@ -491,7 +491,7 @@ class Transaction {
     AWAITING_PREPARE = 1,
     PREPARED = 2,
     AWAITING_COMMIT = 3,
-    COMMITED = 4,
+    COMMITTED = 4,
```

Comment:
This is a public API, so I think it's worth keeping the old name as an alias for the new name, which IIRC can be done with `COMMITED = COMMITTED, // old misspelled name`

Reply:
Added